lua/claudecode/tools/get_diagnostics.lua

```diff
@@ -17,29 +32,78 @@ local function handler(_params) -- Prefix unused params with underscore
     })
   end
 
-  local all_diagnostics = vim.diagnostic.get(0) -- Get for all buffers
+  local log_module_ok, log = pcall(require, "claudecode.logger")
+  if not log_module_ok then
+    return {
+      code = -32603, -- Internal error
+      message = "Internal error",
+      data = "Failed to load logger module",
+    }
+  end
```

Comment:
I don't think that we have to `pcall` this.

Reply:
In that case, I'll have to take a look and do some refactoring. Thanks for the mention 👍